contrib/vmstat.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env drgn
+# Copyright (c) SUSE Linux.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""Dump /proc/vmstat statistics.
+
+   Skip the last enumerator item as it holds the number of enumrator values."""
+
+from drgn.helpers.linux.cpumask import for_each_online_cpu
+from drgn.helpers.linux.kconfig import get_kconfig
+from drgn.helpers.linux.percpu import per_cpu
+
+
+def print_event_line(event, counter):
+    print(f"{event.name:<36} {counter.value_():>16}")
+
+
+print(f'{"Event":<36} {"Count":>16}')
+
+# 1) vm_zone_stat statistics
+print('VM_ZONE_STAT:')
+vm_zone_stat = prog['vm_zone_stat']
+for event in prog.type('enum zone_stat_item').enumerators[:-1]:
+    print_event_line(event, vm_zone_stat[event.value].counter)
+print()
+
+# 2) vm_node_stat statistics
+print('VM_NODE_STAT:')
+vm_node_stat = prog['vm_node_stat']
+for event in prog.type('enum node_stat_item').enumerators[:-1]:
+    print_event_line(event, vm_node_stat[event.value].counter)
+print()
+
+# 3) vm_numa_event statistics are available only if CONFIG_NUMA is enabled
+if 'CONFIG_NUMA' in get_kconfig(prog):
```

Comment:
If you can, it's better to detect this another way rather than by kconfig because lots of distros don't enable `CONFIG_IKCONFIG`. For example, checking for the existence of a variable that only exists if `CONFIG_NUMA` is defined can work:

```suggestion
if "node_subsys" in prog:
```

Reply:
Oh, that's much simpler! I like it.